cmd/run/run.go

```diff
@@ -188,6 +189,22 @@ func isPipe(r io.Reader) bool {
 	return false
 }
 
+// promptFile mirrors the format of .prompt.yml
+type promptFile struct {
+	Name            string `yaml:"name"`
+	Description     string `yaml:"description"`
+	Model           string `yaml:"model"`
+	ModelParameters struct {
+		MaxTokens   *int     `yaml:"maxTokens"`
+		Temperature *float64 `yaml:"temperature"`
+		TopP        *float64 `yaml:"topP"`
+	} `yaml:"modelParameters"`
+	Messages []struct {
+		Role    string `yaml:"role"`
+		Content string `yaml:"content"`
+	} `yaml:"messages"`
+}
```

Comment:
I think you can also add a `testData` section that looks like this:

```yml
testData:
  - input: subatomic particles
  - input: light-speed travel
```

Reply:
For the current use case (running the prompt), testData wouldn't do anything. If we eventually want to add a `gh models eval` command, we'll need to add `testData` and `evaluators` keys.